spec/models/manageiq/providers/kubernetes/container_manager/metrics_capture_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "spec_helper"
+
+describe ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture do
+  before do
+    _guid, _server, @zone = EvmSpecHelper.create_guid_miq_server_zone
+
+    @ems_kubernetes = FactoryGirl.create(:ems_kubernetes, :zone => @zone)
```

Comment:
zone should not be necessary as the factory will create one if needed.

You can probably kill line 5, and then do

```ruby
@ems_kubernetes = FactoryGirl.create(:ems_kubernetes)
@zone = @ems_kubernetes.zone
```